qt/TorrentDelegate.cc

```diff
@@ -578,6 +584,27 @@ void TorrentDelegate::drawTorrent(QPainter* painter, QStyleOptionViewItem const&
 
     painter->setFont(layout.name_font);
     painter->drawText(layout.name_rect, Qt::AlignLeft | Qt::AlignVCenter, layout.nameText());
+    painter->setFont(layout.labels_font);
+
+    // labels
+    int width_used = 0;
+    for (auto label_text : tor.labels())
+    {
+        QRect label_rect = QFontMetrics{ layout.labels_font }.boundingRect(label_text);
+        label_rect.moveTopLeft(layout.labels_rect.topLeft() + QPoint{ width_used, 0 });
+        width_used += label_rect.width();
+
+        // gap between labels
+        width_used += 5;
+
+        auto saved_brush = painter->brush();
```

Comment:
_(nit)_ can be made const? `auto const saved_brush`

Reply:
done